qiskit/mapper/_mapping.py

```diff
@@ -692,8 +699,6 @@ def optimize_1q_gates(circuit):
                                               right_parameters[0],
                                               right_parameters[1],
                                               right_parameters[2])
-                # Evaluate the symbolic expressions for efficiency
-                right_parameters = tuple(map(sympy.N, list(right_parameters)))
```

Comment:
If we simplify a long sequence of u3 gates, is there concern that the expression size will grow? We can test this.